src/cascadia/WindowsTerminal/NonClientIslandWindow.cpp

```diff
@@ -442,11 +442,21 @@ til::rect NonClientIslandWindow::_GetDragAreaRect() const noexcept
             static_cast<float>(_rootGrid.ActualWidth()),
             static_cast<float>(_dragBar.ActualHeight())
         };
+
         const auto clientDragBarRect = transform.TransformBounds(logicalDragBarRect);
+
+        // Make sure to trim the right side of the rectangle, so that it doesn't
+        // hang off the right side of the root window. This normally wouldn't
+        // matter, but UIA will still think its bounds can extend past the right
+        // of the parent HWND.

```

Comment:
Whoa, what does THIS fix?

Reply:
Is this the thing that does nothing for 13631? Should we revert it since it does nothing?

Reply:
It doesn't do _nothing_ - it actually makes the UIA bounding box sane, rather than hanging way off the edge of the window.

Reply:
I can pull that into a separate PR if you want.